wavplayer.cpp

```diff
@@ -211,7 +211,7 @@ std::vector<float> pitchShift(
     
     if(convertToFloat){
       // convert audio data to F32 
-      SDL_BuildAudioCVT(&cvt, srcFormat, 2, srcSampleRate, AUDIO_F32, 2, desiredSampleRate);
+      SDL_BuildAudioCVT(&cvt, srcFormat, 2, srcSampleRate, AUDIO_F32, 2, srcSampleRate);
```

Comment:
keep the source sample rate when converting the audio data to f32 because we set the user-specified sample rate on the audiospec struct that we use for playback. 

if we set it here and on the audiospec struct, that seems to just end up making things sound normal as if the user-specified sample rate was ignored (which I guess makes sense).